src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,10 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T extends Machine> {
+    int NUMBER_OF_MACHINES = 3;
+
+    List<? extends Machine> get();
```

Comment:
use generic type that already bounded. 
```suggestion
    List<T> get();
```